students/thomas_sulgrove/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+##SP_Online_PY210 lesson03 Slicing lab https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/slicing.html
+
+##with the first and last items exchanged
+def exchange_first_last(seq):
+    a_new_sequece = list(seq) #whatever it is, gunna make it a list
+    a_new_sequece[0], a_new_sequece[-1] = a_new_sequece[-1], a_new_sequece[0] #exhange the first and last thing
+    
+    #return it as the same format as was given
+    if isinstance(seq, tuple):
+        return tuple(a_new_sequece)
+    if isinstance(seq, str):
+        return str(''.join(a_new_sequece))
+    
+    
+#with every other item removed
+def remove_other(seq):
+    a_new_sequece = list(seq) #whatever it is, gunna make it a list
+    a_new_sequece = a_new_sequece[::2] #take every other index
+    
+    #return it as the same format as was given
+    if isinstance(seq, tuple):
+        return tuple(a_new_sequece)
+    if isinstance(seq, str):
+        return str(''.join(a_new_sequece))
```

Comment:
there should be no instance checking or changing of data structure here (function is just one line of code)